### PR TITLE
Dsn 11 landing page

### PR DESCRIPTION
**Pull request by pintermate225**
Converting html + css into next.js compatible

### CODE REVIEW

File: DSN/backend/src/main/resources/application.properties
Comment (bela-riboczki-ext) [inline DSN/backend/src/main/resources/application.properties:1]: These changes are not for the Landing Page, I guess. Please revert them.

**Review by bela-riboczki-ext — COMMENTED** [review]
The implementation is looking good, but the Pull Request contains changes which is not for this feature. Please revert those changes.

**Comment by pintermate225** [comment]
Okay i will fix it tomorrow


Bela Riboczki ***@***.***> ezt írta (időpont: 2024. febr.
19., Hét 10:26):

File: DSN/frontend/package.json [hunk 1]
```diff
@@ -9,19 +9,20 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "next": "14.1.0",
     "react": "^18",
     "react-dom": "^18",
-    "next": "14.1.0"
+    "react-router-dom": "^6.22.0"
```
Comment (bela-riboczki-ext) [inline DSN/frontend/package.json]: As we talked about it, this package is not necessary for our solution.

File: DSN/backend/src/main/java/org/gfa/dsn/utils/SecurityConfig.java [hunk 2]
```diff
@@ -53,6 +53,7 @@ public SecurityFilterChain filterChain(HttpSecurity httpSecurity) throws Excepti
             "/v3/api-docs/**",
             "/v3/api-docs.yaml",
             "/api/v1/auth/**",
+
```
Comment (bela-riboczki-ext) [inline DSN/backend/src/main/java/org/gfa/dsn/utils/SecurityConfig.java]: Please remove this unnecessary empty line.

**Review by bela-riboczki-ext — APPROVED** [review]
Please fix one of the changes. Beside of that, it looks good for merge.

**Review by yamenchaban — APPROVED** [review]

File: DSN/backend/src/main/java/org/gfa/dsn/utils/SecurityConfig.java (hunk 2 above)
Comment (pintermate225) [reply DSN/backend/src/main/java/org/gfa/dsn/utils/SecurityConfig.java]: Done

**Review by bela-riboczki-ext — CHANGES_REQUESTED** [review]
We found out there is some errors on the frontend with this version. 
First of all, the background.jpg is missing. If you need that file you can grab from [this](https://github.com/MatejFrnka/social-network/commit/78a52534fbf618c6633c0a223979fca3916e8b32) commit.

After putting the background file to the correct location and adjusting the location in the css, the following error message pops up when I try to run it:

Unhandled Runtime Error
Error: [next-auth]: `useSession` must be wrapped in a <SessionProvider />

Please fix these issues. If you need assistance, you can contact me or @yamenchaban. We are available for assisting you in the frontend development.

**Review by bela-riboczki-ext — APPROVED** [review]

**Pull request merged**